backend/sessions.ts

```diff
@@ -156,4 +177,21 @@ export const getSortedQueue = (code: string) => {
       return 0;
     });
   });
+}
+
+export const addItemToQueue = (code: string, item: Item) => {
+  const session = sessions[code];
+  if (!session) {
+    return;
+  }
+  session.queue.push(item);
+
+  console.log("session", session);
+  if (!session.currentlyPlaying) {
+    if (playerInstances[session.hostId]) {
+      getAndRemoveNextVideoFromSession(code);
+    }
+  }
```

Comment:
This code block is not supposed to be in here yet, because the functionality it depends on is added in fc832e7e04409313df4a5fc4896b7b82693996a5.